Idrall/Check.idr

```diff
@@ -139,12 +158,20 @@ mkEnv ((x, e) :: ctx) =
         (IsA t) => let v = VNeutral t (NVar x) in
                        (x, v) :: env)
 
+-- TODO check is there some FP magic for this
```

Comment:
Hmm, I can't come up with anything directly, for what FP magic would do the same.